tests/diaCatalogSourceSelector.py

```diff
@@ -72,7 +72,7 @@ def makeMatches(self, refCat, srcCat, nSrc):
             refSrc.set("r_flux", 10**(-0.4*18))
             refSrc.set("resolved", False)
             refSrc.set("photometric", True)
-            refSrc.set("coord", coord)
+            refSrc.setCoord(coord)
 
```

Comment:
On second thought, I'm going to leave it for now.  I'm actually deferring the removal of those setters and getters for a while, and part of the problem is that I know I need to find a better syntax to replace them, and I don't have that yet.  The only alternative that would work right now in this case is really verbose, which just reemphasizes that problem.